records_mover/utils/concat_files.py

```diff
@@ -18,10 +20,23 @@ def readable(self) -> bool:
         return True
 
     def readall(self) -> bytes:
-        return b''.join(f.read(-1) for f in self._files)
+        out = b''
```

Comment:
You can take [this particular write up i found](https://www.guyrutenberg.com/2020/04/04/fast-bytes-concatenation-in-python/) with a grain of salt, but it matches my general understanding that concatenating plain strings can be slow. Swapping in a `bytearray` should behave the same with better performance. I guess you might need to cast it to a bytes at the very end to achieve the right types. Anyway, may not make a difference here, but if we end up reading a lot of big files, possible that it could.

Reply:
Ah, thanks - I hadn't seen that write-up!  

My expectation would be that the only thing that might call readall() would be Pandas, which is already pretty doomed by the time it's dealing with files large enough to split up.  Still makes sense to increase the threshold of doom, though!